tests/assembly/inline-asm-avx.rs

```diff
@@ -0,0 +1,23 @@
+// assembly-output: emit-asm
+// compile-flags: --crate-type=lib
+// compile-flags: --target x86_64-unknown-linux-gnu
+// needs-llvm-components: x86
+
+#![feature(portable_simd)]
+
+use std::simd::Simd;
+use std::arch::asm;
+
+#[target_feature(enable = "avx")]
+pub unsafe fn convert(a: *const f32) -> Simd<f32, 8> {
+    let b: Simd<f32, 8>;
+    // CHECK: vbroadcastss
```

Comment:
Please write the check so that it asserts that `ymm_reg` is in fact `ymm{something}`. I'm neutral on the other part of the instruction.